configure.ac

```diff
@@ -2118,7 +2118,7 @@ then
 	case $ac_cv_prog_cc_g in
 	yes)
 	    if test "$Py_DEBUG" = 'true' ; then
-		OPT="-g $PYDEBUG_CFLAGS -Wall"
+		OPT="-g $PYDEBUG_CFLAGS -Wall -Wno-unused-label"
```

Comment:
Do we need this?

Reply:
No. I'm a Git idiot. It snuck back in somehow.